packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3784,285 +3544,73 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Mute the audio for a meeting
-   * @returns {Promise} resolves the data from muting audio {mute, self} or rejects if there is no audio set
+   * Shorthand function to join AND set up media
+   * @param {Object} options - options to join with media
+   * @param {JoinOptions} [options.joinOptions] - see #join()
+   * @param {MediaDirection} [options.mediaOptions] - see #addMedia()
+   * @returns {Promise} -- {join: see join(), media: see addMedia()}
    * @public
    * @memberof Meeting
+   * @example
+   * joinWithMedia({
+   *  joinOptions: {resourceId: 'resourceId' },
+   *  mediaOptions: {
+   *    localTracks: { microphone: microphoneTrack, camera: cameraTrack }
+   *   }
+   * })
    */
-  public muteAudio() {
-    if (!MeetingUtil.isUserInJoinedState(this.locusInfo)) {
-      return Promise.reject(new UserNotJoinedError());
-    }
-
-    // @ts-ignore
-    if (!this.mediaId) {
-      // Happens when addMedia and mute are triggered in succession
-      return Promise.reject(new NoMediaEstablishedYetError());
-    }
+  public joinWithMedia(
+    options: {
```

Comment:
do you know if lobby scenario we are able to send media , any thoughts on join and joinWithMedia 

2